### PR TITLE
Fallback to sponsor name if logo is missing

### PR DESCRIPTION
**Pull request by daytonn**
Instead of rendering an empty string, we are now falling back to the SponsorName component
@kand @collin

### CODE REVIEW

File: elements/campaign-display/components/campaign-display-root.test.js [hunk 1]
```diff
@@ -283,6 +284,11 @@ describe('<campaign-display> <CampaignDisplayRoot>', () => {
         let types = subject.props.children.map((c) => c.type);
         expect(types).to.not.contain(Logo);
       });
+
+      it('renders the sponsor name', () => {
+        let types = subject.props.children.map((c) => c.type);
+        expect(types).to.contain(SponsorName);
```
Comment (collin) [inline elements/campaign-display/components/campaign-display-root.test.js]: This test betrays a desire for enzyme ;)

**Comment by collin** [comment]
👍

File: elements/campaign-display/components/campaign-display-root.test.js (hunk 1 above)
Comment (daytonn) [inline elements/campaign-display/components/campaign-display-root.test.js]: totally

**Comment by kand** [comment]
👍

**Pull request merged**